dev-environment/nussknacker/opt/prinz-sample/base-application.conf

```diff
@@ -0,0 +1,82 @@
+storageDir: ./storage
+storageDir: ${?STORAGE_DIR}
+
+dbFilePath: ${storageDir}"/db"
+dbFilePath: ${?DB_FILE_PATH}
+
+dbName: "db"
+dbName: ${?DB_NAME}
+
+dbUser: "sa"
+dbUser: ${?DB_USER}
+
+dbPassword: ""
+dbPassword: ${?DB_PASSWORD}
+
+dbUrl: "jdbc:hsqldb:hsql://localhost/db;sql.syntax_ora=true"
+dbUrl: ${?DB_URL}
+
+dbDriver: "org.hsqldb.jdbc.JDBCDriver"
+dbDriver: ${?DB_DRIVER}
+
+dbConnectionTimeout: 30000
+dbConnectionTimeout: ${?DB_CONNECTION_TIMEOUT}
+
+jdbcServerEnabled: true
+jdbcServerEnabled: ${?JDBC_SERVER_ENABLED}
+
+jdbcServer: {
+  enabled: ${jdbcServerEnabled}
+  dbFilePath: ${dbFilePath}
+  dbName: ${dbName}
+  user: ${dbUser}
+  password: ${dbPassword}
+}
+
+db {
+  url: ${dbUrl}
+  driver: ${dbDriver}
+  password: ${dbPassword}
+  user: ${dbUser}
+  connectionTimeout: ${dbConnectionTimeout}
+}
+
+commentSettings: {
+  matchExpression: "(issues/[0-9]*)"
+  link: "https://github.com/TouK/nussknacker/$1"
+}
+
+developmentMode: ${?DEVELOPMENT_MODE}
+
+attachmentsPath: ${storageDir}"/attachments"
+
+proxyUrl: "http://localhost:8081"
+proxyUrl: ${?PROXY_URL}
+
+grafanaUrl: ${proxyUrl}"/grafana"
+grafanaUrl: ${?GRAFANA_URL}
+
+#TODO: Figure out the defaults. It's tricky part, because influxUrl and grafanaUrl can point to same location, but one link is used
```

Comment:
TODO shouldn't be merged - open an isssue or solve in this commit if this is yours, remove if unuesed

Reply:
This TODO comes from Nussknacker. Let's keep it here for reference and update configs when they do the fix.